README.md

```diff
@@ -95,19 +97,21 @@ For ClamAV, macOS users can do the following:
 - `source ~/.bash_profile`
 - `freshclam` (installs virus definitions)
 
-Both the front-end (`/web-client`) and API (`/web-api`) share code that exists in `/shared`. Before you can run either, you need to run `npm install` inside the `shared` directory.
+Both the front-end (`/web-client`) and API (`/web-api`) share code that exists in `/shared`. Before you can run either, you need to run `npm install` inside the top-level directory.
```

Comment:
Much of the judge's dashboard is inaccessible to keyboard navigation in Firefox or Safari. Messages cannot be selected, `View All Messages` cannot be selected, and the entire `Trial Sessions` sidebar cannot be selected. Please ensure that keyboard-based navigation permits access to all functionality.

Reply:
@waldoj I suspect some of the concern here was due to the main navigation not being tab-able. That should be fixed with a refactor to use buttons instead of links.

For the remaining links on the page (and this includes ANY link in the application), we will likely continue to run into this problem as there is a - bug? - in the way FF and Safari work on Macs, where links do not automatically get a tab index. This IS a setting you can change in both the browser and system-wide.

I did update navigation so that those are tab-able in all cases. For some of the remaining links, I'm curious if we can assume someone needing this functionality will already have the appropriate settings enabled. Thoughts?

Here's a SO on the subject for further reading: https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox

Reply:
I'm pretty sure this came up maybe six months ago, and somebody (@kkoskelin, maybe?) made the same point then — that nothing is being done to _prevent_ keyboard-based navigation, and the HTML is clean, and that this is just a matter of modifying browser preferences. Makes sense to me.